generated/nifake/nifake_service.cpp

```diff
@@ -29,6 +29,24 @@ namespace nifake_grpc {
       output->Add(item != VI_FALSE);
     }
   }
+
+  template <typename T1>
+  void NiFakeService::CopyEnumValues(const std::string& input, T1* output, const std::map<char, std::int32_t>& enum_map)
+  {
+    for (auto item : input)
+    {
+      auto it = enum_map.find(item);
```

Comment:
8,16, and 32-bit integer values never get mappings generated for them. Remove this `enum_map` from the argument list and simplify the implementation.

Reply:
removed this